README.md

```diff
@@ -6,61 +6,99 @@ To enable/disable modules, update and change the HTTPD installation then there i
 ```
 <VirtualHost *:8080>
-    Include /etc/apache2/sites-enabled/mgmt.conf
-    ### ADD MORE HOSTS BELOW THIS LINE
+
+RewriteEngine on
+RewriteCond %{HTTP:Upgrade} websocket [NC]
+RewriteCond %{HTTP:Connection} upgrade [NC]
+RewriteRule /management/(.*) ws://internal-unity-mc-alb-hzs9j-1269535099.us-west-2.elb.amazonaws.com:8080/$1 [P,L]
+<Location "/management/">
+    ProxyPass http://internal-unity-mc-alb-hzs9j-1269535099.us-west-2.elb.amazonaws.com:8080/
+    ProxyPassReverse http://internal-unity-mc-alb-hzs9j-1269535099.us-west-2.elb.amazonaws.com:8080/
+    ProxyPreserveHost On
+    FallbackResource /management/index.html
+</Location>
 
 </VirtualHost>
 ```
 
-They will be added as additional config files below the comment line. The httpd task is then restarted to allow the 
-config to then take effect.
+Live checking of the "current" configuration may be accomplished with `write_site.py` in a local environment:
+```
+% DEBUG=yes UNITY_PROJECT=btlunsfo UNITY_VENUE=dev11  python write_site.py
+<VirtualHost *:8080>
 
-There is currently no way to remove files or fix a broken config other than mounting the EFS mount into an EC2 server and making changes.
-To do this you will need to edit the security group to allow access to the EC2 box and then install the EFS utils.
+RewriteEngine on
+RewriteCond %{HTTP:Upgrade} websocket [NC]
+RewriteCond %{HTTP:Connection} upgrade [NC]
+RewriteRule /management/(.*) ws://internal-unity-mc-alb-hzs9j-1269535099.us-west-2.elb.amazonaws.com:8080/$1 [P,L]
+<Location "/management/">
+    ProxyPass http://internal-unity-mc-alb-hzs9j-1269535099.us-west-2.elb.amazonaws.com:8080/
+    ProxyPassReverse http://internal-unity-mc-alb-hzs9j-1269535099.us-west-2.elb.amazonaws.com:8080/
+    ProxyPreserveHost On
+    FallbackResource /management/index.html
+</Location>
 
-## Manually adding a file/template
+</VirtualHost>
 
-One can execute the httpdmanager lambda function directly with the following json syntax:
+```
 
+This repository configures only one virtualhost (both open and close directives), but others may be added. This can be accomplished by simply adding more SSM parameters:
 ```
-{
-  "filename": "example-extension",
-  "template": "SSLProxyEngine On\nProxyPreserveHost On\n\nProxyPass \/hub https:\/\/jupyter.us-west-2.elb.amazonaws.com:443\/hub\/\nProxyPassReverse \/hub https:\/\/jupyter.us-west-2.elb.amazonaws.com:443\/hub\/"
+resource "aws_ssm_parameter" "managementproxy_openvirtualhost" {
+  name  = "/unity/${var.project}/${var.venue}/cs/management/proxy/configurations/001-openvirtualhost"
+  type  = "String"
+  value = <<-EOT
+  <VirtualHost *:8080>
+EOT
 }
 
+resource "aws_ssm_parameter" "managementproxy_closevirtualhost" {
+  depends_on = [aws_ssm_parameter.managementproxy_openvirtualhost]
+  name       = "/unity/${var.project}/${var.venue}/cs/management/proxy/configurations/100-closevirtualhost"
```

Comment:
do we want to label this as 100-closevirtualhost8080, since we may have multiple ones in the future, and want to distinguish by port?